app/explore/recordings/[slug]/page.tsx

```
@@ -0,0 +1,118 @@
+  const stillListed = await prisma.recording.findFirst({
+    where: { ...publicRecordingWhere(), id: listing.id },
+    select: { id: true },
+  });
+  if (!stillListed) notFound();
+
+  return (
+    <div className="container mx-auto max-w-5xl px-4 py-10 grid gap-8 lg:grid-cols-[1.6fr_1fr]">
+      <div className="space-y-6">
+        <div className="aspect-video rounded-xl bg-muted flex items-center justify-center overflow-hidden">
+          {listing.previewClipUrl ? (
+            // eslint-disable-next-line jsx-a11y/media-has-caption
```

ESLint

[error] Definition for rule 'jsx-a11y/media-has-caption' was not found.

(jsx-a11y/media-has-caption)